app/assets/javascripts/admin/addon/templates/admin-plugins-show-schema.gjs

```diff
@@ -0,0 +1,35 @@
+import { hash } from "@ember/helper";
+import { LinkTo } from "@ember/routing";
+import RouteTemplate from "ember-route-template";
+import icon from "discourse/helpers/d-icon";
+import { i18n } from "discourse-i18n";
+import Editor, {
+  SCHEMA_MODES,
+} from "admin/components/schema-theme-setting/editor";
+
+export default RouteTemplate(
+  <template>
+    <div class="customize-themes-show-schema__header row">
```

Comment:
The class names in this file is still referencing `themes` which is not correct since this is the plugins route.